services/agent/src/simcore_service_agent/info/_consumer.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+
+from ._constants import HOST, INFO_REQUEST, PORT
+
+_8_BYTES = 8 * 1024
```

Comment:
you mean ``_8_KB`` right?

Reply:
no longer being used, but will consider for the future